client/asset/btc/btc.go

```diff
@@ -1385,6 +1387,96 @@ func (btc *baseWallet) Info() *asset.WalletInfo {
 	return btc.walletInfo
 }
 
+func (btc *baseWallet) startTxHistoryDB() error {
+	dir, err := os.Open(btc.walletDir)
+	if err != nil {
+		return fmt.Errorf("error opening wallet directory: %w", err)
+	}
+	defer dir.Close()
+
+	entries, err := dir.Readdir(0)
+	if err != nil {
+		return fmt.Errorf("error reading wallet directory: %w", err)
+	}
+
+	pattern := regexp.MustCompile(`^txhistory-(.+)\.db$`)
+
+	var dbPath string
+	for _, entry := range entries {
+		if !entry.IsDir() {
+			continue
+		}
```

Comment:
Is this right? Don't we want to skip directories?

Reply:
The DB is stored in a directory.